plugin.offtictor/samples/suscripciones.json

```diff
@@ -0,0 +1,409 @@
+{
+	"subscriptions": 
+	[
+		{
+			"id": "feed/51de655fa201dead6f002d87",
+			"title": "#OpEuribor",
+			"categories": 
+			[
+				
+			],
+
+			"sortid": "51de655fa201dead6f002d87",
+			"firstitemmsec": "1489330973000",
+			"url": "http://opeuribor.es/feed/",
+			"htmlUrl": "http://opeuribor.es",
+			"iconUrl": "//s.theoldreader.com/system/uploads/feed/picture/5125/d9b8/bd92/793c/0c00/icon_9a74.ico"
+		},
+
+		{
+			"id": "feed/51de655fa201dead6f002d92",
+			"title": "15MpaRato",
+			"categories": 
+			[
+				
+			],
+
+			"sortid": "51de655fa201dead6f002d92",
+			"firstitemmsec": "1489574800000",
+			"url": "http://15mparato.wordpress.com/feed/",
+			"htmlUrl": "https://15mparato.wordpress.com",
+			"iconUrl": "//s.theoldreader.com/system/uploads/feed/picture/511d/64eb/e721/ec74/d303/icon_c6df.ico"
+		},
+
+		{
+			"id": "feed/51de6561a201dead6f002dbb",
+			"title": "9 Sant Vicen\u00e7",
+			"categories": 
+			[
+				
+			],
+
+			"sortid": "51de6561a201dead6f002dbb",
+			"firstitemmsec": "1408531853000",
+			"url": "http://www.blogger.com/feeds/1587207914235716365/posts/default",
+			"htmlUrl": "http://www.nousv.com/",
```

Comment:
doesn't exist

Reply:
samples folder is for a guide to develop.
I'll remove it the next release